nuxt_src/components/parts/TheHeader.vue

```diff
@@ -158,21 +175,29 @@
                   {{ $t('cfp') }}
                 </nuxt-link>
               </div>
-            <!--  ログイン前 ここまで -->
-            <!--  ログイン後 ここから -->
-            <!-- <div class="function_item-userIcon">
-              <div class="userMenu">
-                <div class="function_userInfo js-accordion">
-                  <div class="function_userIcon" style="background-image: url('/img/dummy/icon-user2.jpg')"></div>
-                  <p class="function_userName">ポプテピピック氏</p>
+              <!--  ログイン前 ここまで -->
+              <!--  ログイン後 ここから -->
+              <div v-if="isLoggedOut" class="function_item-userIcon">
```

Comment:
あーここが`isLoggeIn`かもですね
```suggestion
              <div v-if="isLoggedIn" class="function_item-userIcon">
```